scripts/bump-kotlin-nightlies.main.kts

```diff
@@ -66,4 +51,56 @@ fun getLatestVersion(url: String, prefix: String? = null): String {
   }
 }
 
+fun runCommand(vararg args: String): String {
+  val builder = ProcessBuilder(*args)
+      .redirectError(ProcessBuilder.Redirect.INHERIT)
+  val process = builder.start()
+  val output = process.inputStream.bufferedReader().readText().trim()
+  val ret = process.waitFor()
+  if (ret != 0) {
+    throw Exception("command ${args.joinToString(" ")} failed:\n$output")
+  }
+  return output
+}
+
+fun runCommand(args: String): String {
+  return runCommand(*args.split(" ").toTypedArray())
+}
```

Comment:
This needs to be in stdlib 😄

Reply:
so true 💯 

There's [this lib](https://github.com/lordcodes/turtle) but didn't look at it too much.

Reply:
Yeaaa and importing the dependency is more lines than copy/pastting `runCommand` everywhere....